src/EditorFeatures/CSharpTest/RemoveUnusedVariable/RemoveUnusedVariableTests.cs

```diff
@@ -205,5 +205,108 @@ void Method()
     }
 }");
         }
+
+        [WorkItem(20987, "https://github.com/dotnet/roslyn/issues/20987")]
```

Comment:
The PR description doesn't mention that it fixes this bug, so it probably won't get closed automatically when you merge.